db/migrate/20150212100000_import_cropstore_db.rb

```diff
@@ -475,9 +475,6 @@ def change
 
       create_table "plant_lines", id: false, force: :cascade do |t|
         t.text "plant_line_name",    primary: true
-        t.text "genus",              default: "unspecified", null: false
```

Comment:
Unfortunately this won't work - we need a separate migration which takes care of this DDL (this also goes for the following two changes). Modifying the bootstrap migration will make it impossible to carry these changes through to the production DB since the production DB does not actually use the bootstrap migration.